Mixpanel/Mixpanel.m

```diff
@@ -1667,16 +1667,17 @@ - (void)notificationController:(MPNotificationViewController *)controller wasDis
     };
 
     if (status && controller.notification.callToActionURL) {
-        MixpanelDebug(@"%@ opening URL %@", self, controller.notification.callToActionURL);
-        BOOL success = [[UIApplication sharedApplication] openURL:controller.notification.callToActionURL];
+        [controller hideWithAnimation:NO completion:^{
```

Comment:
Yes you are totally correct. Sorry for the confusion about the iOS 9 URL changes. I would very much prefer to dismiss the controller with an animation. From a UX perspective I think it's rather jarring to dismiss with no animation. 

If we could dismiss like before based on the success, I would be satisfied merging this today.